domains/User.js

```diff
@@ -53,7 +56,7 @@ class User extends EventEmitter {
 		this._handleMatchRoomClose = this._handleMatchRoomClose.bind(this);
 
 		this.producer.on('message', this._handleProducerMessage);
-		this.producer.once('close', this._handleProducerClose);
+		this.producer.on('close', this._handleProducerClose);
```

Comment:
A connection is short lived, a producer is long lived, that `once()` registration was a bug 😓